resources/profiles/Anker/machine/fdm_machine_common.json

```diff
@@ -110,7 +108,7 @@
     ],
     "silent_mode": "0",
     "single_extruder_multi_material": "1",
-    "use_relative_e_distances": "1",
+    "use_relative_e_distances": "0",
```

Comment:
Relative e is strongly recommended.
I suggest dont change this.

ref: https://github.com/SoftFever/OrcaSlicer/issues/1857

Reply:
put back as suggested